automatic/filezilla/tools/chocolateyInstall.ps1

```diff
@@ -9,7 +9,7 @@ $packageArgs = @{
   file64         = Get-Item $toolsPath\*_x64.exe
   silentArgs     = '/S'
   validExitCodes = @(0, 1223)
-  softwareName   = 'FileZilla Client*'
+  softwareName   = 'FileZilla 3*'
```

Comment:
Agreed. @AdmiringWorm mentioned as part of the PR review that this is likely something that we want to bring out as part of the update.ps1 script, but that this could be achieved in a separate scope of work.